charts/multicluster-controlplane/templates/pvc.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{ .Values.pvcName }}
```

Comment:
we don't need to configure pvc name, I think we need configure `resources.requests.storage`,  `storageClassName ` and `selector`

you may refer to https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims

Reply:
so, in values.yaml, you may have a configuration for pvc, like:

```
pvc:
   storageCapacity: 1Gi
   storageClassName:""
  selector: {}
```